uber/templates/preregistration/preregbase.html

```diff
@@ -239,6 +238,7 @@
                         $.field('amount_extra').on('change', makeBadgeMatchExtra);
                     {% endif %}
                 }
+                togglePrices();
```

Comment:
The call to `togglePrices()` has been moved inside the `if ($(BADGE_TYPES.row).size()) {` conditional. Was that intentional?

`togglePrices()` internally also manipulates `REG_TYPES`, so I would guess we would want to call `togglePrices()` even if `BADGE_TYPES` isn't used?